internal/model/model.go

```diff
@@ -1222,6 +1218,7 @@ nextSub:
 		TempNamer:     defTempNamer,
 		TempLifetime:  time.Duration(m.cfg.Options().KeepTemporariesH) * time.Hour,
 		CurrentFiler:  cFiler{m, folder},
+		MtimeRepo:     db.NewVirtualMtimeRepo(m.db, folderCfg.ID),
```

Comment:
Sorry to revive this, but I'd like some clarification.
This line means the MtimeRepo is set to something (i.e. non-nil), hence line [145](https://github.com/syncthing/syncthing/pull/1804/files#diff-9d41e06c64fc85e225f438897b68f92fR145) (and following) will always use the db?
I don't think this is intentional?